examples/platforms/cc1352/CMakeLists.txt

```diff
@@ -75,6 +76,7 @@ target_compile_definitions(openthread-cc1352
         ${OT_PLATFORM_DEFINES}
     PRIVATE
         ${OT_PRIVATE_DEFINES}
+        "MBEDTLS_CONFIG_FILE=\"mbedtls-config.h\""
```

Comment:
Why added this?

Reply:
MBEDTLS_CONFIG_FILE needs to point to the proper config header when building platform-specific crypto drivers. Suggestions on how best to support this?

Reply:
It seems the `openthread-cc1352` target needs to include mbedtls headers, then we should expect that 
```cmake
target_link_libraries(openthread-cc1352
    PRIVATE mbedtls
)
```
should do all we need, and we can also remove `${OT_PRIVATE_DEFINES}` here (this seems only needed by mbedTLS headers), if we make `${OT_PRIVATE_DEFINES}` PUBLIC definitions of mbedTLS targets.